third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -416,6 +416,30 @@ func TestAccBigQueryDataTable_sheet(t *testing.T) {
 	})
 }
 
+func TestAccBigQueryDataTable_bigtable(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckBigQueryTableDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccBigQueryTableFromBigtable(context),
```

Comment:
Looks like this doesn't match the function declaration, causing this error:
`google-beta/resource_bigquery_table_test.go:432:13: undefined: testAccBigQueryTableFromBigtable`

Reply:
oops, thanks!
Should be fixed now.